src/stcal/ramp_fitting/ols_fit.py

```diff
@@ -3514,7 +3514,7 @@ def calc_opt_sums(rn_sect, gain_sect, data_masked, mask_2d, xvalues, good_pix):
 
     # For those pixels that are all False, set to sentinel value of -1
     fnz[c_mask_2d.sum(axis=0) == 0] = -1
-
+    c_mask_2d[0, 0] = False
```

Comment:
What does this do?

Reply:
I have no idea. I have no memory of changing that line of code.

Reply:
This line is not in the current version of ols_fit.py on stcal github master, so I don't think it belongs here. Might've been an errant, unindented edit of a file in your branch that then got included in your commits to the PR?  At any rate, you need to get rid of it (somehow).